### ISSUE TITLE
add a donation link to readme

### ISSUE THREAD

**Issue by peterbeck** [skx/puppet-summary#78]
Hi Steve,

I really enjoy puppet-summary and I'd gladly send you a donation.
Wouldn't it be nice to have some instructions how to support you financially ?
I can't send you thousands of pounds, but some appreciation... and you really deserve it

Thank you so much,
Cheers Peter

**Comment by skx** [comment]
Don't get me wrong money is nice, but I write/update/share things here as a hobby.

I'm happy if people say nice things now and again, be it issues like this, or emails telling me how using some of "my stuff" helped them in some way.

If you're super-keen then I did setup the sponsorship stuff on a few repositories, if you're particularly keen that's probably the sanest way to go, though I had that more in mind for companies/groups using some of my software rather than individuals.  

Click on the red heart at the top of this repository and that'll walk you through the process (I'm hazy on the details; I've received zero sponsorships so far.)  Otherwise there's also a link to [an amazon wishlist](https://steve.fi/donate/) at the same place.  I'll update that this evening, now you've made me think of it.  After all it seems increasingly unlikely I'll be seeing family in-person this year!

In short the though is nice, and I appreciate it, but it isn't necessary.  If I ever get some sponsorship(s) I'll spend the money on whisky and cakes.  Always good for a celebration!

**Comment by skx** [comment]
* [wishlist link](http://www.amazon.co.uk/exec/obidos/registry/2LTKRAY9O22LI/ref=wl_s_3/202-8341772-7932661) updated to be more current.
* sponsorship "heart" enabled on the repository.

I think that's enough :)
